layers/+emacs/helpful/packages.el

```diff
@@ -0,0 +1,50 @@
+;;; packages.el --- helpful layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Johnson Denen <johnson@johnsons-macbook-pro.local>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;; See the Spacemacs documentation and FAQs for instructions on how to implement
+;; a new layer:
+;;
+;;   SPC h SPC layers RET
+;;
+;;
+;; Briefly, each package to be installed or configured by this layer should be
+;; added to `helpful-packages'. Then, for each package PACKAGE:
+;;
+;; - If PACKAGE is not referenced by any other Spacemacs layer, define a
+;;   function `helpful/init-PACKAGE' to load and initialize the package.
+
+;; - Otherwise, PACKAGE is already referenced by another Spacemacs layer, so
+;;   define the functions `helpful/pre-init-PACKAGE' and/or
+;;   `helpful/post-init-PACKAGE' to customize the package as it is loaded.
+
```

Comment:
Above helpful message regarding layer usage should be removed.

Reply:
Thanks for this catch.